### PR TITLE
Chore: Fixes PhantomJs by adding polyfills for fetch and AbortController

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by dprokop — APPROVED** [review]
Great!

File: contribute/style-guides/e2e.md [hunk 1]
```diff
@@ -150,6 +150,15 @@ describe('List test', () => {
 
 ## Debugging PhantomJS image rendering
 
+### Common Error
```
Comment (dprokop) [inline contribute/style-guides/e2e.md]: Thanks for adding this! This is very helpful, already trying it against https://github.com/grafana/grafana/pull/21587

File: contribute/style-guides/e2e.md [hunk 2]
```diff
@@ -150,6 +150,15 @@ describe('List test', () => {
...
+The most common error with PhantomJs image rendering is when a PR introduces an import that has functionality that's not supported by PhantomJs. To quickly identify which new import causes this you can use a tool like `es-check`.
+   
+1. Install es-check
```
Comment (dprokop) [inline contribute/style-guides/e2e.md]: Maybe instead if suggesting installing, just suggest `npx es-check ...` ?

File: contribute/style-guides/e2e.md [hunk 3]
```diff
@@ -150,6 +150,15 @@ describe('List test', () => {
...
+2. Run > es-check es5 '/public/build/*.js'
```
Comment (dprokop) [inline contribute/style-guides/e2e.md]:
```suggestion
2. Run > es-check es5 './public/build/*.js'
```

Dot was missing, it failed when I tried it with this path

File: contribute/style-guides/e2e.md (hunk 2 above)
Comment (dprokop) [inline contribute/style-guides/e2e.md]:
```suggestion
```

File: contribute/style-guides/e2e.md (hunk 3 above)
Comment (dprokop) [inline contribute/style-guides/e2e.md]:
```suggestion
2. Run  `npx es-check es5 './public/build/*.js'`
```

File: contribute/style-guides/e2e.md [hunk 4]
```diff
@@ -150,6 +150,15 @@ describe('List test', () => {
...
+3. Check the output for files that break es5 compatibility.
```
Comment (dprokop) [inline contribute/style-guides/e2e.md]:
```suggestion
2. Check the output for files that break es5 compatibility.
```

File: contribute/style-guides/e2e.md [hunk 5]
```diff
@@ -150,6 +150,15 @@ describe('List test', () => {
...
+4. Lazy load the failing imports if possible.
```
Comment (dprokop) [inline contribute/style-guides/e2e.md]:
```suggestion
3. Lazy load the failing imports if possible. 
```

File: contribute/style-guides/e2e.md (hunk 2 above)
Comment (hugohaggmark) [reply contribute/style-guides/e2e.md]: Yes, awesome suggestion!

**Pull request merged**